.github/workflows/nodejs.yml

```diff
@@ -44,29 +53,33 @@ jobs:
       run: npm -g install npm@6
 
     - run: npm install -g yarn
+
     - run: yarn bootstrap
+    
+    - run: yarn depcheck || (echo 'Please add missing dependencies to appropriate package'; false)
```

Comment:
I think we're probably better off without the explicit custom error message here; I think the existing error is clear enough and the method for including it is kind of hacky.  I only put an explicit custom error message on yarncheck because otherwise it's just not clear at all with that one.

(If `yarn --frozen-lockfile` worked like it was supposed to, and I didn't need the custom test, then that would probably be clear and I could omit the custom error... but it doesn't, so yeah.)

Reply:
Sure, I'll remove that.

Reply:
Simulated an error [and it looks ok](https://github.com/trufflesuite/truffle/runs/7254092415).